lib/pubsub/index.js

```diff
@@ -84,6 +84,11 @@ function PubSub(options) {
 
   var config = {
     baseUrl: process.env.PUBSUB_HOST || 'https://pubsub.googleapis.com/v1',
+    proto: {
+      baseUrl: 'pubsub.googleapis.com',
+      service: 'pubsub',
+      apiVersion: 'v1'
```

Comment:
I think it would be nice to try and have the `proto` object piggy back off of the `baseUrl` (or vice-versa) that way if and when we update the api version we're not doing it in multiple places.